app/assets/stylesheets/mixins/course/forum/_forum_post.scss

```diff
@@ -0,0 +1,34 @@
+$border_color: #ddd;
+$shadow_color: rgba(0, 0, 0, 0.075);
+
+@mixin forum-post {
+  .user {
+    float: left;
+    margin-right: 1em;
+    text-align: right;
+    width: 7em;
+  }
+
+  .contents {
+    border: 1px solid $border_color;
```

Comment:
Name of variable `border_color` should be written in all lowercase letters with hyphens instead of underscores